dace/transformation/interstate/__init__.py

```diff
@@ -5,4 +5,3 @@
 from .fpga_transform_sdfg import FPGATransformSDFG
 from .gpu_transform_sdfg import GPUTransformSDFG
 from .sdfg_nesting import NestSDFG, InlineSDFG
-from .double_buffering import DoubleBuffering
```

Comment:
Add newline at end of file, so that git and formatting plugins don't complain.

Reply:
According to my file there is already a newline...